docs/docsite/rst/scenario_guides/guide_gce.rst

```diff
@@ -254,3 +254,43 @@ Note that use of the "add_host" module above creates a temporary, in-memory grou
 in the 'new_instances' group, if so desired.  Any sort of arbitrary configuration is possible at this point.
 
 For more information about Google Cloud, please visit the `Google Cloud website <https://cloud.google.com>`_.
+
+Migration Guides
+----------------
+
+gce.py -> gcp_compute_instance.py
+`````````````````````````````````
+We're encouraging everyone to move from the `gce` module to the
```

Comment:
A note here about when (which Ansible version) `gce.py` was deprecated and when `gcp_compute_instance` was created would be helpful for future maintenance.